samples/subsys/ipc/ipm_psoc6/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _ipm-psoc6-sample:
+
+Sample mailbox application
+##########################
+
+Overview
+********
+
+The :ref:`cy8ckit_062_wifi_bt` board has two core processors (Cortex-M4
+and Cortex-M0+). This sample application uses a mailbox to send messages
+from Cortex-M4 processor core (client) to Cortex-M0+ (server).
+
+Requirements
+************
+
+- :ref:`cy8ckit_062_wifi_bt` board
+
+Building and Running
+********************
+
+$ cd samples/subsys/ipc/ipm_psoc6
+$ mkdir build
+$ cd build
+$ cmake -G"Eclipse CDT4 - Ninja" ..
+$ ninja
+
+After flashing PSoC6 open two serial terminals (minicom, putty, etc.) and
+connect to the board with the following settings:
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Reset the board and the following message will appear on the corresponding
+serial port:
+- Server UART
+
+.. code-block:: console
```

Comment:
Contents of the code-block must be indented (to be noticed as part of the code block):

```
.. code-block:: console

   ***** Booting Zephyr OS zephyr-v1.13.0-1623-gb7c7bb89c1 *****
   PSoC6 IPM Server example has started
   Received 0 via IPC
   ...
   Received 99 via IPC
   ...
```